.github/workflows/integration-tests.yml

```diff
@@ -73,10 +73,10 @@ jobs:
         run: |
           cd python
           python3 -m pip install --upgrade pip
-          python3 -m pip install cmake==3.24
-          python3 -m pip install ninja
-          python3 -m pip install --no-build-isolation -vvv '.[tests]'
-          python3 -m pip install pytest-xdist
+          python3 -m pip install cmake==3.24 ninja pytest-xdist
+          sudo apt-get update
+          sudo apt-get install ccache clang lld
```

Comment:
I think I had tried that but it didn't work. I believe we need to install clang directly on the CI machines

Reply:
It seems to be working for me!

Reply:
oh cool! I think it used to complain about `sudo apt-get` but looks like the new CI machine takes it.